pom.xml

```diff
@@ -502,5 +502,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Todo remove this dependency -->
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
+        </dependency>
```

Comment:
What do we need this for though?

Reply:
We use it everywhere in Slimefun currently. All the Validate's are from commons lang, however Spigot now removed it so here we are.

To keep this change small and quick, I told Jeff to just add this for now and we can refactor as a fast follow.

Reply:
Ah, `Validate`, I see